contracts/NiftyFootballTradingCardEliteBlindPack.sol

```diff
@@ -116,10 +120,10 @@ contract NiftyFootballTradingCardEliteBlindPack is Ownable, Pausable {
         return tokenId;
     }
 
-    function _takePayment() internal {
+    function _takePayment(uint256 _numberOfCards) internal {
         // any trapped ether can be withdrawn with withdraw()
         totalPurchasesInWei = totalPurchasesInWei.add(msg.value);
```

Comment:
With the refund/overspend this value isnt accurate anymore

Reply:
fixed